crates/viewer/re_renderer/src/renderer/lines.rs

```diff
@@ -9,9 +9,7 @@
 //! The quads are oriented and spanned in a vertex shader.
 //!
 //! It is tempting to use instancing and store per-instance (==quad) data in a instance-stepped vertex buffer.
-//! However, GPUs are notoriously bad at processing instances with a small batch size as
-//! [various](https://gamedev.net/forums/topic/676540-fastest-way-to-draw-quads/5279146/)
-//! [people](https://gamedev.net/forums/topic/702292-performance-fastest-quad-drawing/5406023/)
+//! However, at least historically GPUs are notoriously bad at processing instances with a small batch size as
```

Comment:
This is change intentional?

Reply:
yeah, the other links broke and ci complained about it. Also I read by now that modern GPUs no longer have this issue (... on the GameDev Tech Slack, so can't post a source), do doesn't hurt to tune it down